src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -397,6 +398,21 @@ public void init(AdaptorContext context) {
         this.queryStrings);
     log.log(Level.CONFIG, "opentext.displayUrl.objAction: {0}",
         this.objectActions);
+    // Validate the Display URLs.
+    try {
+      URI uri = getDisplayUrl("", 0L);
+      new ValidatedUri(uri.toString()).logUnreachableHost();
+    } catch (URISyntaxException e) {
+      throw new InvalidConfigurationException("Invalid Display URL", e);
+    }
+    for (Map.Entry<String, String> entry : queryStrings.entrySet()) {
```

Comment:
If you only use the keys, why get the entrySet instead of the keySet? Also, I think you need the union of the keySets for queryStrings and objectActions. Or are you arguing that only the query strings need to be checked, as the most obvious place for URL syntax errors (the objectActions being [a-zA-Z]+ identifiers)?